data/docmaps/sample_docmap_for_85111.json

```diff
@@ -449,22 +478,58 @@
           "doi": "10.1101/2022.11.08.515698",
           "url": "https://www.biorxiv.org/content/10.1101/2022.11.08.515698v3",
           "versionIdentifier": "3"
-        },
+        }
+      ],
+      "next-step": "_:b5",
+      "previous-step": "_:b3"
+    },
+    "_:b5": {
+      "actions": [
         {
-          "type": "reply",
-          "doi": "10.7554/eLife.85111.1.sa1"
+          "participants": [],
+          "outputs": [
+            {
+              "identifier": "85111",
+              "versionIdentifier": "2",
+              "type": "preprint",
+              "doi": "10.7554/eLife.85111.2",
+              "license": "http://creativecommons.org/licenses/by/4.0/"
+            }
+          ]
+        }
+      ],
+      "assertions": [
+        {
+          "item": {
+            "type": "preprint",
+            "doi": "10.7554/eLife.85111.2",
+            "versionIdentifier": "2"
+          },
+          "status": "manuscript-published"
+        }
+      ],
+      "inputs": [
+        {
+          "type": "preprint",
+          "doi": "10.1101/2022.11.08.515698",
+          "url": "https://www.biorxiv.org/content/10.1101/2022.11.08.515698v3",
+          "versionIdentifier": "3"
         },
         {
           "type": "review-article",
-          "doi": "10.7554/eLife.85111.1.sa2"
+          "doi": "10.7554/eLife.85111.2.sa1"
         },
         {
           "type": "review-article",
-          "doi": "10.7554/eLife.85111.1.sa3"
+          "doi": "10.7554/eLife.85111.2.sa2"
         },
         {
           "type": "evaluation-summary",
-          "doi": "10.7554/eLife.85111.1.sa4"
+          "doi": "10.7554/eLife.85111.2.sa3"
+        },
+        {
+          "type": "reply",
+          "doi": "10.7554/eLife.85111.2.sa4"
         }
       ],
       "previous-step": "_:b3"
```

Comment:
It might be easier to add my feedback here about this particular line, I think perhaps based on the step structure this could be

`"previous-step": "_:b4"`

Reply:
Ah that is right. Thank you!